src/components/tabs/_tabs.scss

```diff
@@ -90,57 +91,78 @@
           padding-bottom: govuk-spacing(2);
         }
 
-        @include govuk-if-ie8 {
-          // IE8 doesn't support `box-shadow` so add an actual border that is
-          // modified to indicate state.
-          border-top: 4px solid transparent;
+        // Since borders are joined with a  45 degree angle bevel 'mitre joint',
+        // we can't set the top border color.
+        //
+        // +-------------+
+        // | \=========/ |
+        // |   +-----+   |
+        // |   | Tab |   |
+        // +---+     +---+
+        //
+        // To get around this we float a border above the tab
+        // so that it'll be 90 degree angle 'butt joint'.
+        //
+        // +---+-----+---+
+        // |   |=====|   |
+        // |   +-----+   |
+        // |   | Tab |   |
+        // +---+     +---+
+        //
+        &:after {
+          content: "";
+          position: absolute;
+          top: -1px;
+          right: 0;
+          left: 0;
+          border-width: 0;
+          border-style: solid;
+          border-color: transparent;
         }
 
         &:hover {
-          box-shadow: inset 0 4px 0 0 govuk-colour("light-blue");
+          // Remove outline from unenhanced focus state
+          outline: none;
+          // Remove styles from unenhanced focus state
+          box-shadow: none;
 
-          @include govuk-if-ie8 {
+          &:after {
+            border-top-width: 4px;
             border-top-color: govuk-colour("light-blue");
           }
         }
 
         &--selected {
-          $top-border-width: 1px;
-          $top-border-adjustment: 1px;
-
-          @include govuk-if-ie8 {
-            // IE8 already has a top border so no adjustment is needed
-            $top-border-width: 4px;
-            $top-border-adjustment: 0;
-          }
-
-          margin-top: - govuk-spacing(1);
+          margin-top: - (govuk-spacing(1) - 1px);
           margin-bottom: -1px;
 
-          // 1px is compensation for border (otherwise we get a 1px shift)
-          padding-top: govuk-spacing(3) - $top-border-adjustment;
+          padding-top: govuk-spacing(3) - 2px;
           padding-right: govuk-spacing(4) - 1px;
           padding-bottom: govuk-spacing(3) + 1px;
           padding-left: govuk-spacing(4) - 1px;
 
-          border-top: $top-border-width solid transparent;
-          border-right: 1px solid $govuk-border-colour;
-          border-left: 1px solid $govuk-border-colour;
+          border: 1px solid $govuk-border-colour;
+          border-bottom: 0;
 
           color: $govuk-text-colour;
           background-color: govuk-colour("white");
-          box-shadow: inset 0 4px 0 0 $govuk-link-colour;
           text-decoration: none;
 
-          @include govuk-if-ie8 {
+          &:after {
```

Comment:
I've had to use `:after` instead of `::after` since Internet Explorer 8 does not support the non deprecated selector, anyone know why our build steps don't convert these automatically? 🤔

Reply:
I *think* we currently only do prefixing with postcss. We'd probably want this: https://github.com/timelsass/postcss-pseudo-element-colons

Reply:
The 'oldie' plugin already supports this, but for some reason [it's disabled](https://github.com/alphagov/govuk-frontend/blob/master/tasks/gulp/compile-assets.js#L78).

Reply:
Looks like we may have turned this off so that the radios/checkboxes component can fallback easier... https://github.com/alphagov/govuk-frontend/blame/fdde74e1887b81e3419cd82dc55dc793398117d3/tasks/gulp/compile-assets.js

We might be able to guard this CSS with the IE8 mixins and turn this back on...